locales/en-US/strings.properties

```diff
@@ -153,10 +153,15 @@ topsites_form_title_label=Title
 topsites_form_title_placeholder=Enter a title
 topsites_form_url_label=URL
 topsites_form_url_placeholder=Type or paste a URL
+topsites_form_image_label=Custom Image
+topsites_form_image_placeholder=Paste a URL here
+topsites_form_image_button=Preview
+topsites_form_image_enable_button=Use a custom image…
```

Comment:
I ended up renaming this in a WIP if we were to have landed strings sooner. But sounds like we'll do it Monday.

```diff
 topsites_form_url_placeholder=Type or paste a URL
+topsites_form_use_image_link=Use a custom image…
+topsites_form_image_label=Custom Image
+topsites_form_image_placeholder=Paste a URL here
+# LOCALIZATION NOTE (topsites_form_*_button): These are verbs/actions.
+topsites_form_preview_button=Preview
 topsites_form_add_button=Add
 topsites_form_save_button=Save
 topsites_form_cancel_button=Cancel
 topsites_form_url_validation=Valid URL required
+topsites_form_image_validation=Image failed to load. Try a different URL.
 
```